projects/observability/src/shared/components/status-display/status-display.component.ts

```diff
@@ -0,0 +1,26 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { IconSize } from '@hypertrace/components';
+
+@Component({
+  selector: 'ht-status-display',
+  template: ` <div class="status" *ngIf="this.statusCode">
+    <ng-container *ngIf="this.statusCode | htStatusDisplayIcon: this.status as icon">
+      <ht-icon [icon]="icon" size="${IconSize.ExtraSmall}"></ht-icon>
+    </ng-container>
+    <span class="text">{{ this.statusCode | htStatusDisplayText: this.statusMessage }}</span>
+  </div>`,
+  styleUrls: ['./status-display.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StatusDisplayComponent {
+  @Input()
+  public statusCode?: number | string;
+
+  @Input()
+  public status?: string;
+
+  @Input()
+  public statusMessage?: string;
+
+  // TODO: Add display for different styles if required
```

Comment:
@arjunlalb @adisreyaj 
We have this component as gray background and since we want this to be used at the status table cell renderer (which has different styling according to the status), how do we wanna approach this?

1. Should we replace that styling with this new one
2. Or add that styling as a config here as well?

Reply:
We should use a single style. I recommend using the existing style. After consulting with design team, we can switch it over to the new one or keep the old one.

Reply:
I agree on keeping the same styling throughout the product and yeah, the existing one seems better to me as well. But let's wait for @adisreyaj to comment on this since I think he has more context around that.

I think, We can leave this here as it is until we get final confirmation.